util/util_test.go

```diff
@@ -44,3 +48,57 @@ func TestFilterFileNames(t *testing.T) {
 		}
 	}
 }
+
+func TestSweetConfigDir(t *testing.T) {
+	t.Run("correct directory is returned", func(t *testing.T) {
```

Comment:
When should I use the `t.Run` function, and when should I not? It's not too important, but it would be nice to have some rules and consistency with how the tests look.